src/thunderbots/software/ai/hl/stp/evaluation/indirect_chip.h

```diff
@@ -0,0 +1,115 @@
+#include "ai/world/world.h"
+#include "geom/point.h"
+
+/**
+ * The indirect chip and chase target evaluation function returns a target point for the
+ * chipper and chaser. 
+ * Other related evaluation functions are also included in this file.
+ */
+namespace Evaluation
+{
+    typedef Point Vector2;
+    template <size_t N>
+    using Poly     = std::array<Vector2, N>;
+    using Triangle = Poly<3>;
+
+    // Minimum area of chip target triangle
+    const double MIN_CHIP_TRI_AREA(0.5);
+    // Minimum edge length of chip target triangle
+    const double MIN_CHIP_TRI_EDGE_LEN(0.8);
+    // Percentage of distance to center of triangle to return as target
+    const double CHIP_CHERRY_POWER_DOWNSCALE(0.85);
+    // Maximum power the robot can chip the ball at without malfunctions
+    const double MAX_CHIP_POWER(8.0);
+    // Closest distance to edge of field that the robot could chip and chase to
+    const double CHIP_TARGET_AREA_INSET(0.3);
```

Comment:
These values were originally defined using `DoubleParam` type in the old code base and it consisted of a default, min, and max value. 

Is there an equivalent data type in our current code base? Here I'm just using the default values for now.

Reply:
Yes the equivalent now is the `Parameter` class in `software/util/parameter/dynamic_parameter.*`. #160 has a bit more detailed instruction that you can follow to replace these.